analysis/configs/efficiencies.md

```diff
@@ -76,12 +76,12 @@ electronMaxHoverE_Barrel:         0.005
 electronMaxHoverE_Endcap:      0.005
 electronMaxDetaSeedBarrel:      0.1
 electronMaxDetaSeedEndcap:   0.1
-electronMaxChargedIsoBarrel:    0.1
```

Comment:
Please don't override the default config, but rather create a new one and point to it in your scripts.

Reply:
Hi @jniedzie , I have implemented your suggestions. 
Sure, I will create smaller PR's more often in future. :)